### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by webdevmission**
HELP!!!
No syntax error in code but when rendered only blank page shown.
Any idea what's could be the problem? Cheers!

### CODE REVIEW

**Comment by Berenike-Kuhli** [comment]
did it ever work? did you see the example app from the boilerplate?

File: src/global/components/Layout/index.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const index = () => {
+  return <div className="bg-d-medium"></div>;
```
Comment (Berenike-Kuhli) [inline src/global/components/Layout/index.jsx]: not sure, but maybe you need to enter {children} as placeholder here? like
`function DefaultLayout({ children, className }) {
    return (
        <div className="bg-dark h-auto w-100 ${className}">
            { children }
            <NavBar/>
        </div>
    );
}`

File: src/App/index.jsx [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
-  {
-    path: "/",
-    element: <App />,
-  },
-]);
+const App = () => {
+  return <Dashboard />;
+};
+
+App();
```
Comment (Berenike-Kuhli) [inline src/App/index.jsx:8]: what does this line do? why do you call App again?

Comment (Berenike-Kuhli) [reply src/App/index.jsx:8]: I wonder whats the difference as my functions read
`function App()  {
return (
some content
);
}
export default App`

**Review by Berenike-Kuhli — COMMENTED** [review]
no idea if any of this is the cause of your trouble

when did it break? or did it never work in the first place? If you never saw the example app, there might be something wrong with the setup, something not properly installed

**Comment by webdevmission** [comment]
everything fixed now. thx for your contribution

**Pull request closed without merging**